app/components/Settings/ProfileImageChange.jsx

```diff
@@ -0,0 +1,173 @@
+var React = require('react')
+var AWS = require('aws-sdk')
+var cfgAWS = require('../../../config/aws.json')
+AWS.config.update(cfgAWS.cred);
+AWS.config.region = cfgAWS.region;
+var Dropzone = require('react-dropzone')
+var ProfileImage = require('./ProfileImage.jsx')
+
+var ProfileImageChange = React.createClass({
+  
+  sizeMini : 100,
+  sizeMidi : 750,
+
+  getInitialState : function() {
+    return {
+      showDropzone: false,
+      srcImageMini: this.props.srcMini,
+      srcImageMidi: this.props.srcMidi
+    };
+  },
+  
+  changeValue: function (event) {
+    this.setState({
+       showDropzone: true
+    }); 
+  },
+  
+  imageS3url : function(cfg, objKey) {
+        return 'https://s3.'+ cfg.region +'.amazonaws.com/' + cfg.bucketName + '/'+objKey;
+  },
+  
+  multiple : false,
+  
+  resizeImage : function(file, maxWidth, maxHeight, cb) {
+      
+      var img = document.createElement("img"),
+          reader = new FileReader();  
+      
+      reader.onload = function(eR) {
+          
+          img.onload = function(eI) {
+          
+              var canvas = document.createElement("canvas");
+              var ctx = canvas.getContext("2d");
+              ctx.drawImage(this, 0, 0);
+      
+              var width = this.width;
+              var height = this.height;
+              var dx = 0;
+              var dy = 0;
+              
+              if (width > height) {
+                if (width > maxWidth) {
+                  width *= maxHeight / height;
+                  height = maxHeight;
+                  dx = (width - maxWidth) / -2;
+                }
+              } else {
+                if (height > maxHeight) {
+                   height *= maxWidth / width;
+                   width = maxWidth;
+                   dy = (height - maxHeight) / -2;
+                }
+              }
+              canvas.width = maxWidth;
+              canvas.height = maxHeight;
+              var ctx = canvas.getContext("2d");
+              
+              ctx.drawImage(this, dx, dy, width, height);
+      
+              
+              canvas.toBlob(cb);
```

Comment:
Można dodać jakąś małą bibliotekę polyfill, żeby uzupełnić ten brak. Znalazłem 2, które możnaby sprawdzić i ewentualnie zastosować:
https://github.com/blueimp/JavaScript-Canvas-to-Blob
https://github.com/eligrey/canvas-toBlob.js/